Project/src/main/resources/templates/upload.html

```diff
@@ -20,10 +20,41 @@
             </th>
         </thead>
         <tbody>
-            <tr th:each="file: ${files}">
-                <td th:text="${file.id}"></td>
-                <td th:text="${file.fileName}"></td>
-                <td><a th:href="@{/getfile/{id}(id=${file.id})}">Download</a></td>
+            <tr th:each="article: ${articles}">
```

Comment:
Why is the accepting tool added to upload.html? They're supposed to have two different actions. editors can't submit files. There should be a separate view for this.